addons/project/models/project_project.py

```diff
@@ -104,6 +104,7 @@ def _set_favorite_user_ids(self, is_favorite):
     user_id = fields.Many2one('res.users', string='Project Manager', default=lambda self: self.env.user, tracking=True)
     alias_id = fields.Many2one(help="Internal email associated with this project. Incoming emails are automatically synchronized "
                                     "with Tasks (or optionally Issues if the Issue Tracker module is installed).")
+    alias_domain_id = fields.Many2one(inherited=True, default=lambda self: self.env.company.alias_domain_id)
```

Comment:
You don't need to override the alias_domain_id field. We need to load alias_id in the form view.